ydb/library/actors/wilson/wilson_uploader.cpp

```diff
@@ -136,28 +141,37 @@ namespace NWilson {
             void Bootstrap() {
                 Become(&TThis::StateWork);
 
+                if (MaxSpansPerSecond == 0) {
+                    ALOG_WARN(WILSON_SERVICE_ID, "max_spans_per_second should be greater than 0, changing to 1");
+                    MaxSpansPerSecond = 1;
+                }
+                if (MaxSpansInBatch == 0) {
+                    ALOG_WARN(WILSON_SERVICE_ID, "max_spans_in_batch shold be greater than 0, changing to 1");
+                    MaxSpansInBatch = 1;
+                }
+
                 TStringBuf scheme;
                 TStringBuf host;
                 ui16 port;
                 if (!TryGetSchemeHostAndPort(CollectorUrl, scheme, host, port)) {
-                    LOG_ERROR_S(*TlsActivationContext, WILSON_SERVICE_ID, "Failed to parse collector url (" << CollectorUrl << " was provided). Wilson wouldn't work");
+                    ALOG_ERROR(WILSON_SERVICE_ID, "Failed to parse collector url (" << CollectorUrl << " was provided). Wilson wouldn't work");
                     Become(&TThis::StateBroken);
                     return;
                 } else if (scheme != "grpc://" && scheme != "grpcs://") {
-                    LOG_ERROR_S(*TlsActivationContext, WILSON_SERVICE_ID, "Wrong scheme provided: " << scheme << " (only grpc:// and grpcs:// are supported). Wilson wouldn't work");
+                    ALOG_ERROR(WILSON_SERVICE_ID, "Wrong scheme provided: " << scheme << " (only grpc:// and grpcs:// are supported). Wilson wouldn't work");
                     Become(&TThis::StateBroken);
                     return;
                 }
                 Channel = grpc::CreateChannel(TStringBuilder() << host << ":" << port,
                                               scheme == "grpcs://" ? grpc::SslCredentials({}) : grpc::InsecureChannelCredentials());
                 Stub = NServiceProto::TraceService::NewStub(Channel);
 
-                LOG_INFO_S(*TlsActivationContext, WILSON_SERVICE_ID, "TWilsonUploader::Bootstrap");
+                ALOG_INFO(WILSON_SERVICE_ID, "TWilsonUploader::Bootstrap");
```

Comment:
If we have an indicator of working/erroneous Wilson, we can reduce level to DEBUG here.

Reply:
Do we have one?